plugins/arcgis/web-app/projects/main/src/lib/arc-event/arc-event.component.html

```diff
@@ -1,45 +1,46 @@
 <div class="arc-event">
   <mat-card appearance="outlined">
     <mat-card-header>
-      <mat-card-title>
-        Events 
-      </mat-card-title>
+      <mat-card-title>Events</mat-card-title>
       <mat-card-subtitle>Change the ArcGIS layers each MAGE event sends its observations to.</mat-card-subtitle>
     </mat-card-header>
     <mat-card-content>
-  <section class="arc-event-sync">
-    <div class="arcEvents" *ngIf="!model.events.length">
-      <div class="arcEvent">
-        There are no events synchronizing to ArcGIS layers.
-      </div>
-    </div>
-    <div class="arcEvents" *ngFor="let event of model.events">
-      <ng-container>
-        <div class="arcEvent">
-          <div class="arcEvent__value">
-            <a href="/#/admin/events/{{event.id}}" target="_blank">{{event.name}}</a>
-          </div>
-          <div class="arcEvent__edit">
-            <button class="edit-button" mat-icon-button (click)="onEditEvent(event)"><mat-icon class="edit-icon" [inline]="true">edit</mat-icon></button>
-          </div>
-        </div>
-        <div class="featureLayers" *ngIf="!event.layers.length">
-          <div class="featureLayer_value">
-            This event is not synchronizing to any ArcGIS layers.
-          </div>
+      <section class="arc-event-sync">
+        <div *ngIf="!model.events.length">
+          There are no events synchronizing to ArcGIS layers.
+          <mat-divider></mat-divider>
         </div>
-        <div class="featureLayers" *ngFor="let featureLayer of event.layers">
-          <div class="featureLayer__value">
-            {{layerDisplay(featureLayer)}}
-          </div>
-        </div>
-      </ng-container>
-    </div>
-  </section>
-</mat-card-content>
-</mat-card>
 
-<ng-template #editEventDialog let-data>
+        <mat-list>
+          <ng-container *ngFor="let event of model.events; let last = last">
+            <mat-list-item class="arcEvent">
+              <div class="arcEvent__info">
+                <a href="/#/admin/events/{{event.id}}" target="_blank">{{event.name}}</a>
```

Comment:
Lets use a list action to link users to the external feature service.  See here for an example: https://github.com/ngageoint/mage-server/blob/switch-httpClient-to-esri-library/plugins/arcgis/web-app/projects/main/src/lib/arc-layer/arc-layer.component.html#L30-L32